crates/turborepo-lib/src/run/scope/change_detector.rs

```diff
@@ -138,9 +138,14 @@ impl<'a> GitChangeDetector for ScopeChangeDetector<'a> {
             }
             PackageChanges::Some(packages) => {
                 debug!(
-                    "determined {} packages changed: {packages:?}",
-                    packages.len()
+                    "{} packages changed: {:?}",
+                    packages.len(),
+                    &packages
+                        .iter()
+                        .map(|x| x.name.to_string())
+                        .collect::<Vec<String>>()
```

Comment:
sort also? or is packages already sorted?

Reply:
Nope, sorting is good here. Packages are stored in a hash set so just iterating over them won't yield a stable order.